app/scripts/views/settings/avatar_change.js

```diff
@@ -87,6 +87,8 @@ function ($, Cocktail, FormView, AvatarMixin, SettingsMixin, Template,
       var self = this;
       var defer = p.defer();
       var file = e.target.files[0];
+      var account = self.getSignedInAccount();
+      self.logAccountImageChange(account);
```

Comment:
:+1:  This code only executes if they do select a file, although it could be incompatible. There's also the chance they will bail during the cropper step. Are these concerns?

Tangential, but it would be neat to record how the cropper is used, e.g. rotates, zooms, etc.